libraries/Microsoft.Bot.Connector/Attachments.cs

```diff
@@ -66,152 +62,168 @@ public Attachments(ConnectorClient client)
-                ex.Response = new HttpResponseMessageWrapper(_httpResponse, _responseContent);
-                if (_shouldTrace)
+
+                ex.Request = new HttpRequestMessageWrapper(httpRequest, requestContent);
+                ex.Response = new HttpResponseMessageWrapper(httpResponse, responseContent);
+                if (shouldTrace)
                 {
-                    ServiceClientTracing.Error(_invocationId, ex);
+                    ServiceClientTracing.Error(invocationId, ex);
                 }
-                _httpRequest.Dispose();
-                if (_httpResponse != null)
+
+                httpRequest.Dispose();
+
+                if (httpResponse != null)
                 {
-                    _httpResponse.Dispose();
+                    httpResponse.Dispose();
                 }
+
                 throw ex;
             }
+
             // Create Result
-            var _result = new HttpOperationResponse<AttachmentInfo>();
-            _result.Request = _httpRequest;
-            _result.Response = _httpResponse;
+            var result = new HttpOperationResponse<AttachmentInfo>();
+            result.Request = httpRequest;
+            result.Response = httpResponse;
+
             // Deserialize Response
-            if ((int)_statusCode == 200)
+            if ((int)statusCode == 200)
             {
-                _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+                responseContent = await httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
                 try
                 {
-                    _result.Body = Rest.Serialization.SafeJsonConvert.DeserializeObject<AttachmentInfo>(_responseContent, Client.DeserializationSettings);
+                    result.Body = Rest.Serialization.SafeJsonConvert.DeserializeObject<AttachmentInfo>(responseContent, Client.DeserializationSettings);
                 }
                 catch (JsonException ex)
                 {
-                    _httpRequest.Dispose();
-                    if (_httpResponse != null)
+                    httpRequest.Dispose();
+                    if (httpResponse != null)
                     {
-                        _httpResponse.Dispose();
+                        httpResponse.Dispose();
                     }
-                    throw new SerializationException("Unable to deserialize the response.", _responseContent, ex);
+
+                    throw new SerializationException("Unable to deserialize the response.", responseContent, ex);
                 }
             }
-            if (_shouldTrace)
+
+            if (shouldTrace)
             {
-                ServiceClientTracing.Exit(_invocationId, _result);
+                ServiceClientTracing.Exit(invocationId, result);
             }
-            return _result;
+
+            return result;
         }
 
+#pragma warning disable SA1625 // Element documentation should not be copied and pasted
```

Comment:
>#pragma warning disable SA1625  [](start = 0, length = 31)

Please add a more detailed param descriptions and remove these pragmas.

Reply:
Done! Docs updated -- removed `#pragma warning disable SA1625`